version/VERSION

```diff
@@ -1 +1 @@
-2.3.5
+2.3.6-alpha1
```

Comment:
Does it matter if this eventual version may not be released? I.e. should we do `2.4.0-alpha1` if we know the next version will be a minor bump?

No strong feelings other way, just curious.

Reply:
Yeah I agree with doing `v2.3.6`. I feel pretty confident that this provider won't receive any changes worth a minor version bump in the next couple months 👍🏻